`raylib5.c3l` failing to link with `glfw3`

on arch, fedora, and debian linux, `glfw3` is linked with using `-lglfw` instead of `-lglfw3`.
this causes `raylib5.c3l` and `raylib.c3l` to fail to build with the message `/usr/bin/ld: cannot find -lglfw3: No such file or directory`.
it can be fixed by replacing `"linked-libraries" : [ "glfw3" ]` with `"linked-libraries" : [ "glfw" ]`

after more testing,
method: install gcc and `glfw3` (`libglfw3-dev`, `glfw-devel`, etc) in a distrobox container, then run a raylib hello world that uses the library, but use either `glfw` or `glfw3` in the `manifest.json` of `raylib5.c3l`
work means the program ran
fail means the compilation failed with the error `/usr/bin/ld: cannot find -lglfw3: No such file or directory`
|Distro | glfw      | glfw3      |
|---------------| ------------- | ------------- |
|Arch| work | fail |
|Fedora 40| work | fail |
|Debian 12| work | fail |
|Ubuntu 24.04| work | fail |
|OpenSUSE Tumbleweed| work | fail |

also `glfw.c3l` uses `"linked-libraries" : [ "glfw" ]`

running `pkg-config --libs glfw3` also returns `-lglfw`

--- a/libraries/raylib.c3l/manifest.json
+++ b/libraries/raylib.c3l/manifest.json
@@ -14,7 +14,7 @@
     "linux-x64" : {
       "link-args" : [],
       "dependencies" : [],
-      "linked-libraries" : ["raylib", "GLESv2", "glfw3", "c"]
+      "linked-libraries" : ["raylib", "GLESv2", "glfw", "c"]
     },
 	"windows-x64" : {
 		"linked-libraries" : ["raylib", "opengl32", "kernel32", "user32", "gdi32", "winmm", "winspool", "comdlg32", "advapi32", "shell32", "ole32", "oleaut32", "uuid", "odbc32", "odbccp32"],

--- a/libraries/raylib5.c3l/manifest.json
+++ b/libraries/raylib5.c3l/manifest.json
@@ -14,7 +14,7 @@
     "linux-x64" : {
       "link-args" : [],
       "dependencies" : [],
-      "linked-libraries" : ["raylib", "GLESv2", "glfw3", "c"]
+      "linked-libraries" : ["raylib", "GLESv2", "glfw", "c"]
     },
 	"windows-x64" : {
 		"linked-libraries" : ["raylib", "opengl32", "kernel32", "user32", "gdi32", "winmm", "winspool", "comdlg32", "advapi32", "shell32", "ole32", "oleaut32", "uuid", "odbc32", "odbccp32"],

